.gitmodules

```diff
@@ -25,4 +25,4 @@
 	ignore = dirty
 [submodule "src/protocol-next/xdr"]
 	path = src/protocol-next/xdr
-	url = https://github.com/stellar/stellar-xdr-next.git
```

Comment:
I suspect this needs to switch back to the stellar repo?

Reply:
Sure (hence the !removeme commit) - IIUC that's the only way to make the tests pass before the XDR changes are merged.